src/StructuredLogViewer/Controls/BuildControl.xaml.cs

```diff
@@ -190,6 +193,10 @@ public BuildControl(Build build, string logFilePath)
             // Build Log
             var contextMenu = new ContextMenu();
             contextMenu.Opened += ContextMenu_Opened;
+            searchMenuGroup = new() { Header = "Search ..." };
+            copyMenuGroup = new() { Header = "Copy ..." };
+            gotoMenuGroup = new() { Header = "Go to ..." };

```

Comment:
nit: menu groups don't usually end with "...".

Reply:
Any suggestion?
Just `Search`
or `Search >`

Reply:
Looking at menus in other software, the control adds the arrow for you. And your screenshots show the same. So the text would just be e.g. Search.

Reply:
It blends in so well, I didn't even notice.  Nice.